modules/blog/blog-home/blog-home.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useRef } from 'react';
+import classNames from 'classnames';
+import Link from 'next/link';
+import Image from 'next/image';
+
+import { PostInterface } from '../../../pages/blog';
+import { dateFormatting } from '../../../utils/date-formatting';
+import {
+  tiltEffectSettings,
+  getTiltEffectValues,
+} from '../../../utils/get-tilt-effect-values';
+
+import style from './blog-home.module.scss';
+
+interface PostHomeTypes {
```

Comment:
I think we should be using `type` for our React component props/state, like we had been doing [here](https://github.com/kedro-org/kedro-website/blob/519efebd6fcecaf4e7b09422eb9c5b9aa771e70a/modules/shared/media/media.tsx#L7). I think I have may introduced this, so that's my fault.

```suggestion
type PostHome = {
```

Reply:
You can leave as-is for now and I can fix it later. Don't worry about changing it now. It isn't incorrect.

Reply:
ah okay cool will do that